packages/edge-gateway/src/metrics.js

```diff
@@ -233,7 +233,9 @@ export async function metricsGet(request, env, ctx) {
             `nftgateway_requests_per_time_total{gateway="${gw}",le="${msToS(
```

Comment:
We should probably update here as well, although it will require a change in grafana also.

```suggestion
            `nftgateway_successful_requests_per_time_total{gateway="${gw}",le="${msToS(
```